tests/notification.c

```diff
@@ -59,6 +59,24 @@ notification_fail (GObject      *source,
   g_main_context_wakeup (NULL);
 }
 
+static void
+notification_fail_no_error_check (GObject *source,
+                                  GAsyncResult *result,
+                                  gpointer data)
+{
+  XdpPortal *portal = XDP_PORTAL (source);
+  g_autoptr(GError) error = NULL;
+  gboolean res;
+
+  res = xdp_portal_add_notification_finish (portal, result, &error);
+  g_assert_false (res);
```

Comment:
Might be worth checking the error instead, so that it'll print what was the error if it fails:

```suggestion
  g_assert_no_error (error);
```

Reply:
This function checks for an error without checking what the ever actually was.